src/Sparrow/Platform/Posix/SmapsReader.cs

```diff
@@ -327,7 +327,11 @@ private int ReadFromFile(Stream fileStream, int bufferIndex)
                                 ["SharedClean"] = Sizes.Humane(valSharedClean),
                                 ["SharedDirty"] = Sizes.Humane(valSharedDirty),
                                 ["PrivateClean"] = Sizes.Humane(valPrivateClean),
-                                ["PrivateDirty"] = Sizes.Humane(valPrivateDirty)
```

Comment:
Note that you are sending `Rss` here

Reply:
here? it says "Rss" already.  Where should I note this?

Reply:
https://github.com/aviviadi/ravendb/blob/9fa2477b0c8b58b9a1d275cc8d7ef0daadcc5521/src/Sparrow/Platform/Posix/SmapsReader.cs#L326

Reply:
I am not sure I follow. I am sending Rss, yes.  And that is what is written there.

Reply:
can you show the output from windows & linux for these endpoints?